wiki/osu!supporter/en.md

```diff
@@ -1,39 +1,35 @@
 
--   osu!direct (Direct download in-game)
--   Access to multiplayer when using osu!cuttingedge.
--   [Chat Console](/wiki/Chat_Console): Bright yellow name others can see. However, you still see your username as white.
--   [Beatmap listing](https://osu.ppy.sh/p/beatmaplist): Filter beatmap searches by played and unplayed maps.
--   [Skinning](/wiki/Skinning): Some additional skin elements can be used, such as the [main menu](https://osu.ppy.sh/forum/t/96949)'s background.
-    -   Specifically, `menu-background.jpg` (Main Menu); `triangle.jpg` (Triangles); `welcome.wav` (Welcome Sound); `seeya.wav` (Exiting Sound)
--   [Feature Requests](https://osu.ppy.sh/forum/4): Extra votes for feature requests (2 per month over usual 1 per month).
--   [Song Selection](/wiki/Interface): The ability to view beatmap's in your country / friends / mod-specific rankings in-game.
--   Your Profile: <https://osu.ppy.sh/u/>
-    -   You can add a user page to your profile.
-        -   Your user page will not be deleted when your osu!supporter expires.
-    -   The ability to [change your username](https://osu.ppy.sh/p/profile-username/).
-        -   You can still change your username when your osu!supporter expires.
-    -   Increased beatmap submission limits (up to 10 pending beatmaps total).
--   [Performance Ranking](https://osu.ppy.sh/p/pp): Allow "filter to friends" command.
--   *More to come!*
-
-## Business part
-
-### Cost
-
-[Use this currency convertor](https://www.oanda.com/currency/converter/) or [ask Google](https://www.google.com.my/#q=usd+exchange+rate) to check the price on your country rate. The costs are in "USD ($, United States Dollars)" currency.
-
-| Months | Cost {USD$} (against actual overall) |
+- osu!direct (In-game direct download function)
+- Access to [Multi](/wiki/Multi "Multi") when using `osu!cuttingedge`.
+- [Chat Console](/wiki/Chat_Console "Chat Console"): Bright yellow name for others. However, own username is shown as white instead.
+- [Beatmap listing](https://osu.ppy.sh/p/beatmaplist "Beatmap list"): Filter beatmap searches by played and unplayed maps.
+- [Skinning](/wiki/Skinning "Skinning"): Some additional skin elements can be used, such as the [main menu](https://osu.ppy.sh/forum/t/96949)'s background.
+  - Specifically, `menu-background.jpg` (Main Menu); `triangle.jpg` (Triangles); `welcome.wav` (Welcome Sound); `seeya.wav` (Exiting Sound)
+- [Feature Requests](https://osu.ppy.sh/forum/4 "Feature Requests"): Extra votes for feature requests (2 per month over usual 1 per month).
+- [Song Selection](/wiki/Interface "Song Selection in Interface"): The ability to view beatmap's in own country/friends/mod-specific rankings in-game.
+- Profile
+  - A user page can be added to the profile.
+    - The user page will not be deleted when osu!supporter expires.
+  - The ability to [change username](https://osu.ppy.sh/p/profile-username/).
+    - Username can still be changed even when osu!supporter expires.
```

Comment:
hmm, maybe note that this is true if they have not changed their name during the osu!supporter